### DISCUSSION TITLE
Misleading Documentation

### DISCUSSION THREAD

**Discussion by nilsmartel** [helix-editor/helix#7827] [General]
https://docs.helix-editor.com/keymap.html#changes
![Bildschirmfoto 2023-08-04 um 11 57 14](https://github.com/helix-editor/helix/assets/28377948/6af38d24-f2ec-46c4-92ad-91f028e1e3bb)

Here it is implied that `d` would yank the selection, which is, at least for me, not happening

**Comment by pascalkuthe** [comment]
`d` yanks the selection and then deletes it. You can paste it afterwards with `p`
